articles/sentinel/migration-arcsight-historical-data.md

```diff
@@ -39,11 +39,11 @@ Use the lacat utility to export data from ArcSight Logger. lacat exports CEF rec
 
 To export data with the lacat utility:
 
-1. [Download the lacat utility](https://github.com/hpsec/lacat).
+1. [Download the lacat utility](https://github.com/hpsec/lacat). For large volumes of data, Microsoft suggests modifications to the script to be more performant. You can find these modifications [here](http://aka.ms/lacatmicrosoft). 
```

Comment:
'For large volumes of data, we suggest that you [modify the script](http://aka.ms/lacatmicrosoft) for better performance.'